OpenSim/Sandbox/ExampleHopperDevice/buildHopperModel.cpp

```diff
@@ -151,13 +151,17 @@ Model buildHopper() {
     // vastus muscle can wrap. In the future, we will be able to attach wrap
     // objects to PhysicalOffsetFrames; for now, we will use the existing
     // "translation" property.
+    auto patellaFrame = new PhysicalOffsetFrame("patellaFrame",
+        *thigh, SimTK::Transform(linkDistalPoint));
     auto patella = new WrapCylinder();
     patella->setName("patella");
     patella->set_radius(0.08);
     patella->set_length(linkRadius*2.);
-    patella->set_translation(linkDistalPoint);
+    //patella->set_translation(linkDistalPoint);
```

Comment:
I think these commented lines (160 and 162) can be deleted. (We wouldn't want a workshop attendee to uncomment them.)